sources/platform/schedules.md

```diff
@@ -6,79 +6,84 @@ category: platform
+Schedules allow you to run your Actors and tasks at specific times. You schedule the run frequency using [cron expressions](#cron-expressions).
+
+:::note Timezone & Daylight Savings Time
+Schedules allow timezone settings and support daylight saving time shifts (DST).
+:::
+
+There are few ways you can set up and manage your Schedules:
 
-> Schedules allow timezone settings and support daylight saving time shifts (DST).
+* [Apify Console](https://console.apify.com/schedules)
+* [Apify API](https://docs.apify.com/api/v2#/reference/schedules)
+* [JavaScript API client](https://docs.apify.com/api/client/js/reference/class/ScheduleClient)
+* [Python API client](https://docs.apify.com/api/client/python/reference/class/ScheduleClient)
 
-You can set up and manage schedules from
-[Apify Console](https://console.apify.com/schedules)
-and via [API](/api/v2#/reference/schedules)
-(also with the
-[JavaScript](/api/client/js/reference/class/ScheduleClient) and
-[Python](/api/client/python/reference/class/ScheduleClient)
-API clients).
-When scheduling a new Actor or task run, you can override its input settings using a JSON object similarly to when invoking a schedule using the [Apify API](/api/v2#/reference/schedules/).
+When scheduling a new Actor or task run, you can override its input settings using a JSON object similarly to when invoking a schedule using the [Apify REST API](/api/v2#/reference/schedules/).
 
-> In most cases, scheduled events are fired within one second of their scheduled time. <br/>
-> Occasionally, however, runs can be delayed because of a system overload or a server shutting down.
+:::note Events Startup Variability
+In most cases, scheduled events are fired within one second of their scheduled time. <br/>
+Occasionally, however, runs can be delayed because of a system overload or a server shutting down.
+:::
 
-Each schedule can be associated with a maximum of 10 Actors and 10 Actor tasks.
+Each schedule can be associated with a maximum of _10_ Actors and _10_ Actor tasks.
 
 ## Setting up a new schedule
 
 Before setting up a new schedule, you should have the [Actor](./actors/index.mdx) or [task](./actors/running/tasks.md) whose run you want to schedule prepared and tested.
 
-If you are planning to schedule an Actor run, you need to use the Actor before you can schedule any runs. Navigate to the Actor's page in [Apify Store](https://apify.com/store), click the **Try for free** button, then **Start** it with your preferred settings.
+To schedule an Actor, you need to have run it at least once before. To run the Actor, navigate to the Actor's page in [Apify Store](https://apify.com/store), and click the **Try for free** button. This will direct you to Apify Console, where you can configure and initiate the Actor's run with your preferred settings. After this initial run, you can then use Schedules to automate future runs of the Actor
```

Comment:
Question: Not sure if this is true, i think it's more like "having interacted" with the Actor, not necessarily running it.

Maybe this could be just some: Can't find the Actor? We're only showing Actor's you are using....

Reply:
That is true, you just have to interact it, but are there any situation when you would interact with it but not run it? Maybe something along the lines

`To schedule an Actor, you need to interact with it at least once before. An example of such interaction is running ii` and here proceed with text as is to explain how to run an actor?

Reply:
There technically is - eg. saving the input, and not clicking "Start", but it's not very probable. So looking at it like this, let's stay with run :).

Maybe I'd just change the process from "go to store, try for free, run it" to one that would stay within the console?